content/applications/sales/crm/track_leads/resellers.rst

```diff
@@ -0,0 +1,124 @@
+report. To view the partner levels, navigate to :menuselection:`CRM app --> Configuration -->
+Partner Activations`.
+
+Three activation types are created by default in the *CRM* app:
+
+- :guilabel:`Fully Operational`
+- :guilabel:`Ramp-up`
+- :guilabel:`First Contact`
+
+New levels can be added, as needed, and the existing levels can be edited and renamed, if desired.
+To change the active status of an activation, slide the toggle of the desired activation to the
+*inactive* position.
+
+.. figure:: resellers/activations-toggle.png
+   :align: center
+   :alt: The list of default partner activations in the CRM app.
+
+   The list of default Partner Activations in the CRM app. The toggle for First Contact is in the
+   inactive position, while the rest are active.
+
+Partner assignments
+===================
+
+After partner levels and partner activations have been configured, individual partner records can be
+configured.
+
+To update an individual partner record, navigate to :menuselection:`CRM app --> Sales -->
+Customers`, and click the Kanban card for the desired partner to open the customer record.
+
+On the customer record, click the :guilabel:`Partner Assignment` tab.
+
+Click the :guilabel:`Partner Level` field, and select an option from the drop-down menu to assign a
+level. Select a :guilabel:`Partner Activation` type from the drop-down list, if desired. Then, click
+the :guilabel:`Level Weight` field to assign a different level weight, if necessary.
+
+Publish partners
+================
+
+With the Odoo *Website* app installed, a new webpage can be created to display a list of all active
+partners from the *CRM* app. Navigate to :menuselection:`Website app --> Site --> Pages` and click
+:guilabel:`New`. Hover over the :guilabel:`Blank Page` template, and click the :guilabel:`Use this
+template` button to create a new page.
+
+.. image:: resellers/blank-page-template.png
+   :align: center
+   :alt: The create new page pop-up in the Website app.
+
+In the :guilabel:`New Page` pop-up window, enter a :guilabel:`Page Title`, e.g. `Partners`. Then
+click :guilabel:`Create`. In the :guilabel:`Edit` toolbar on the far right of the page, click
+:guilabel:`Save`.
+
+Next, return to :menuselection:`CRM app --> Sales --> Customers`, and click the Kanban card for a
+partner. From that partner's contact form, click the :guilabel:`Go to Website` smart button at the
+top of the page to open that partner's webpage.
+
+Next, click :guilabel:`Edit` at the top-right of the partner's webpage, and use the :doc:`building
+blocks <../../../websites/website/web_design/building_blocks>` to add any additional design
+elements, or information about the partner.
+
+.. tip::
+  A company summary is a useful addition to this page.
+
+After making any necessary changes to the page, click :guilabel:`Save`. At the top of the page,
+slide the :guilabel:`Unpublished` toggle to the active, :guilabel:`Published` position.
```

Comment:
```suggestion
slide the :guilabel:`Unpublished` toggle to the active, :guilabel:`Published` position, if needed.
```
OPTIONAL: adding "if needed" just in case a partner page has already been published in the past... just a suggestion...
Could even reword to say "make sure the page is published, by...etc."